pennylane_lightning/core/src/simulators/lightning_qubit/gates/tests/CMakeLists.txt

```diff
@@ -36,7 +36,8 @@ target_link_libraries(compile_time_tests    lightning_compile_options
                                             lightning_qubit_utils
                                             )
 
-set(TEST_SOURCES    Test_AVXSingleQubitGateHelpers.cpp
+set(TEST_SOURCES    
+                    Test_AVXSingleQubitGateHelpers.cpp
```

Comment:
Let's gradually transition to this new style.
This makes it easier to comment out any test files during development.